Extension/tslint.json

```diff
@@ -1,31 +1,184 @@
 {
     "rules": {
+        "adjacent-overload-signatures": true,
+        "align": true,
+        "array-type": false,
+        "arrow-parens": false,
+        "arrow-return-shorthand": true,
+        "await-promise": false,
+        "ban-comma-operator": true,
+        "binary-expression-operand-order": true,
+        "callable-types": true,
+        "class-name": true,
+        "comment-format": true,
+        "completed-docs": false,
+        "no-console": false,
         "curly": true,
+        "cyclomatic-complexity": false,
+        "deprecation": false,
+        "encoding": true,
+        "eofline": true,
         "ext-variable-name": [
             true,
-            ["class", "pascal"],
-            ["function", "camel"]
+            [
+                "class",
+                "pascal"
+            ],
+            [
+                "function",
+                "camel"
+            ]
         ],
-        "file-header": [true, ".*"],
-        "indent": [true, "spaces", 4],
+        "file-header": [
+            true,
+            ".*"
+        ],
+        "forin": false,
+        "import-spacing": true,
+        "indent": [
+            true,
+            "spaces",
+            4
+        ],
+        "interface-name": false,
+        "interface-over-type-literal": false,
+        "jsdoc-format": false,
+        "label-position": true,
+        "linebreak-style": false,
```

Comment:
This should be true, and git should also make sure our *.ts files are consistent.

Reply:
Do you know how to get "true" to work on Windows? It gives me "Expected linebreak to be 'LF'" on every line. Are you saying my VS Code git settings aren't configured correctly?

Reply:
Sorry, what I mean is that we should be checking out as with the OS default EOL type and checking is as just LF. E.g. `Checkout Windows (or Unix)-style, commit Unix-style`

https://help.github.com/en/articles/dealing-with-line-endings